### DISCUSSION TITLE
How do I statically link openssl in Arrow?

### DISCUSSION THREAD

**Discussion by davesworking** [microsoft/vcpkg#29999] [Q&A]
which file should I modify to have the openssl libs link statically in the Arrow 9 package? static-link-libs.patch?

Thanks!

**Comment by dg0yt** [comment]
If you don't want to use a static triplet, you can use a custom dynamic triplet (file) which sets `VCPKG_LIBRARY_LINKAGE` to `static` for port `openssl`.
https://learn.microsoft.com/en-us/vcpkg/users/triplets#per-port-customization
